src/utils/Constants/AppInfo/AppInfo.js

```diff
@@ -10,6 +10,7 @@ const decimalSeparator = ','
 const thousandsSeparator = '.'
 const minEntropyLength = 192
 const DEFAULT_WALLETS_TO_CREATE = ['btc']
+const ML_COINS_PER_TOKEN = 100000000000
```

Comment:
We generally call it atoms per coin. In the core node we use the term coin for ML and token to mean a user created token. I would keep the name naming to make it clear to people what we mean